### ISSUE TITLE
contributing file

### ISSUE THREAD

**Issue by Sloth-Panda** [srimani-programmer/Breast-Cancer-Predictor#51]
would like to add contributing file to this repo as part of gssoc'21 @srimani-programmer

**Comment by khareyash05** [comment]
May i work on this issue @Sloth-Panda ?

**Comment by srimani-programmer** [comment]
@Sloth-Panda please work on it. please complete on this issue ASAP.

**Comment by Sloth-Panda** [comment]
Yes @srimani-programmer I am making the pr soon  can you please add  "level1/level2" tag to this issue as easy medium tags are not getting considered as per gssoc guideline

**Comment by Hamza3226** [comment]
Hello, I am a GSSOC'21 participant and would like to contribute to this project.

**Comment by sayan9112** [comment]
Hello, I am a GSSOC'21 participant and would like to contribute to this project.

**Comment by honey2158** [comment]
Hello, I am GSSOC'21 participant and I would like to contribute in this project. I will try to implement other models for this project and to improve it's accuracy.

**Comment by Sloth-Panda** [comment]
@srimani-programmer can you please check this pr

**Comment by honey2158** [comment]
What is it's pr?

**Comment by iamansoni** [comment]
Is this issue still needs to be addressed!
If yes, you can assign it to me!
